lib/rules/length-zero-no-unit/README.md

```diff
@@ -1,6 +1,6 @@
-# length-zero-no-unit
+  # length-zero-no-unit
 
-Disallow units for zero lengths.
+Disallow units for zero lengths. This rule ignores lengths within math functions (e.g. `calc`) in favor of the [`function-calc-no-invalid`](../function-calc-no-invalid/README.md) rule.
```

Comment:
Can you please move the "This rule ignores" sentence _to below the example code block_ as we use this documentation pattern in lots of other places ([like so](https://github.com/stylelint/stylelint/tree/master/lib/rules/font-family-name-quotes#font-family-name-quotes)).

Reply:
Hope now is fine